packages/flame/lib/src/sprite_batch.dart

```diff
@@ -206,6 +215,8 @@ class SpriteBatch {
   /// The size of the [atlas].
   Vector2 get size => atlas.size;
 
+  bool get applyFlip => _applyFlip;
```

Comment:
The doc-comment should be on this public getter, instead of the private variable -- because documentation is generated for public fields only.

Reply:
Thanks for pointing that out.